tests/contrib/langchain/conftest.py

```diff
@@ -5,7 +5,7 @@
 
 from ddtrace import Pin
 from ddtrace import patch
```

Comment:
Weird request but while you're at it, can you replace this import with `from ddtrace.contrib.internal.langchain.patch import patch`? Should just be a quick fix to https://github.com/DataDog/dd-trace-py/blob/de4e2347d19fccf842febea123f0fd01fb5cbdb6/tests/contrib/langchain/conftest.py#L94 for consistency with the other tests ❤️